### DISCUSSION TITLE
1ページにまとめて出力

### DISCUSSION THREAD

**Discussion by koriym** [alps-asd/app-state-diagram#108] [Ideas]
インラインリンクだけで、全ての出力をまとめるのはどうでしょうか？

* ダイアグラム（詳細にリンク）
* ディスクリプタIDリスト （表記は今と同じでリンクは詳細リストへ）
* ディスクリプタ詳細リスト（１ページだったものが並びます）
*最後にprofile.xml|json

可搬性が向上します。

**Comment by suzumaze** [comment]
クラスレスCSSフレームワークの比較です。

- github-markdown
![github-markdown](https://user-images.githubusercontent.com/17695387/142816966-7e9ae699-39eb-450e-9187-0be821538a4d.png)

- simplecss
![simplecss](https://user-images.githubusercontent.com/17695387/142816957-7276098b-0d9a-4afc-b9dc-48d4340edd4d.png)

**Reply by koriym** [reply]
現状の `github-markdown` で良さそうですね。
